ui-admin/src/forms/designer/questions/questionTypes.tsx

```diff
@@ -17,3 +17,71 @@ export const questionTypeDescriptions: Record<QuestionType, string> = {
   signaturepad: 'Prompts the participant to sign a form.',
   medications: 'Prompts the participant to choose medications from a list.'
 }
+
+//Returns an object with all possible fields for each question type
+export const baseQuestions: Record<QuestionType, Question> = {
+  checkbox: {
+    type: 'checkbox',
+    name: '',
+    title: '',
+    description: '',
+    isRequired: false,
+    choices: [],
+    showNoneItem: undefined,
+    noneText: undefined,
+    noneValue: undefined,
```

Comment:
Purely a side note, but it's odd to me that dropdown and radio questions do not offer a "none" option.